### ISSUE TITLE
Error report for runjs 2.1.1

### ISSUE THREAD

**Issue by lehembreg** [lukehaas/RunJS#318]
Error:
```Error: The command is disabled and cannot be executed
    at u.quitAndInstall (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:86311)
    at Immediate._onImmediate (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:51310)
    at processImmediate (internal/timers.js:461:21)
```
OS: macOS 10.16

**Comment by lukehaas** [comment]
Thanks, please also provide your debug log.

![Screenshot 2021-08-31 at 10 15 49](https://user-images.githubusercontent.com/741344/131496256-489dad15-26f8-46c1-8d1c-4180958ec8a8.png)

**Comment by lehembreg** [comment]
[
  {
    "path": "/Users/user/Library/Logs/runjs/main.log",
    "lines": [
      "[2021-06-24 01:09:52.862] [info] Checking for update",
      "[2021-06-24 01:09:52.944] [info] Generated new staging user ID: 1e45c275-3fcb-5b47-badc-aba088b89ed7",
      "[2021-06-24 01:09:54.031] [info] Update for version 2.0.0 is not available (latest version: 2.0.0, downgrade is disallowed).",
      "[2021-08-18 20:27:31.286] [info] Checking for update",
      "[2021-08-18 20:27:32.141] [info] Found version 2.1.1 (url: RunJS-2.1.1-universal-mac.zip, RunJS-2.1.1-universal.dmg)",
      "[2021-08-18 20:27:35.911] [info] Downloading update from RunJS-2.1.1-universal-mac.zip, RunJS-2.1.1-universal.dmg",
      "[2021-08-18 20:27:35.963] [info] Checking for update",
      "[2021-08-18 20:27:36.112] [info] Found version 2.1.1 (url: RunJS-2.1.1-universal-mac.zip, RunJS-2.1.1-universal.dmg)",
      "[2021-08-18 20:27:42.278] [info] New version 2.1.1 has been downloaded to /Users/user/Library/Application Support/Caches/runjs-updater/pending/RunJS-2.1.1-universal-mac.zip",
      "[2021-08-18 20:27:42.369] [info] / requested",
      "[2021-08-18 20:27:42.387] [info] /1629314862291-8779.zip requested",
      "[2021-08-18 20:27:42.390] [info] /1629314862291-8779.zip requested by Squirrel.Mac, pipe /Users/gaspardlehembre/Library/Application Support/Caches/runjs-updater/pending/RunJS-2.1.1-universal-mac.zip",
      "[2021-08-18 20:27:54.742] [info] Proxy server for native Squirrel.Mac is closed (was started to download https://github.com/lukehaas/RunJS/releases/download/v2.1.1/RunJS-2.1.1-universal-mac.zip)",
      "[2021-08-31 12:28:36.907] [info]  Checking for update",
      "[2021-08-31 12:28:37.541] [info]  Found version 2.1.3 (url: RunJS-2.1.3-universal-mac.zip, RunJS-2.1.3-universal.dmg)",
      "[2021-08-31 12:28:39.360] [info]  Downloading update from RunJS-2.1.3-universal-mac.zip, RunJS-2.1.3-universal.dmg",
      "[2021-08-31 12:28:39.380] [info]  Cached update sha512 checksum doesn't match the latest available update. New update must be downloaded. Cached: xzCHrvtQqgkvUXI5xUpJuZAiYsJrybKfclKcFEPPDjZzHC5SrQ+QRMLvCvmqZ0le9RxO2B9sOHg3XBeqzmDlYQ==, expected: TfD0IKnb+Q5fbZZjX4+0eLjqJEnFbYfvF7gCdsP4fJcCanr84CLFOktGOgWzoXYBxqtthpGymGu6hw7DAllutg==. Directory for cached update will be cleaned",
      "[2021-08-31 12:28:41.028] [info]  Checking for update",
      "[2021-08-31 12:28:41.573] [info]  Found version 2.1.3 (url: RunJS-2.1.3-universal-mac.zip, RunJS-2.1.3-universal.dmg)",
      "[2021-08-31 12:28:45.793] [info]  New version 2.1.3 has been downloaded to /Users/user/Library/Application Support/Caches/runjs-updater/pending/RunJS-2.1.3-universal-mac.zip",
      "[2021-08-31 12:28:52.826] [warn]  Error: The command is disabled and cannot be executed",
      "    at u.quitAndInstall (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:86311)",
      "    at Immediate._onImmediate (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:51310)",
      "    at processImmediate (internal/timers.js:461:21)",
      "[2021-08-31 12:28:52.828] [error] Error: Error: The command is disabled and cannot be executed",
      "    at u.quitAndInstall (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:86311)",
      "    at Immediate._onImmediate (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:51310)",
      "    at processImmediate (internal/timers.js:461:21)",
      "[2021-08-31 12:28:54.438] [error] Error: Error: The command is disabled and cannot be executed",
      "    at u.quitAndInstall (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:86311)",
      "    at Immediate._onImmediate (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:51310)",
      "    at processImmediate (internal/timers.js:461:21)",
      "[2021-08-31 12:28:56.446] [error] Error: The command is disabled and cannot be executed",
      "    at u.quitAndInstall (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:86311)",
      "    at Immediate._onImmediate (/Applications/RunJS.app/Contents/Resources/app.asar/entry-bundle.js:11:51310)",
      "    at processImmediate (internal/timers.js:461:21)",
      "[2021-08-31 12:28:56.465] [info]  / requested",
      "[2021-08-31 12:28:56.468] [info]  /1630409325795-8076.zip requested",
      "[2021-08-31 12:28:56.468] [info]  /1630409325795-8076.zip requested by Squirrel.Mac, pipe /Users/user/Library/Application Support/Caches/runjs-updater/pending/RunJS-2.1.3-universal-mac.zip",
      "[2021-08-31 12:29:01.678] [info]  Proxy server for native Squirrel.Mac is closed (was started to download https://github.com/lukehaas/RunJS/releases/download/v2.1.3/RunJS-2.1.3-universal-mac.zip)",
      ""
    ]
  }
]

**Comment by lukehaas** [comment]
Thanks for providing this @lehembreg. 
At the time that you saw this error were you attempting to update RunJS?
